src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -530,6 +534,24 @@ function processItem(item, transaction) {
 }
 ```
 
+#### Retrieving a Transaction
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.getCurrentHub().getScope().getTransaction()`. This function will return a `Transaction` in case there is a running Transaction otherwise it returns `undefined`. If you are using our Tracing integration by default we attach the Transaction to the Scope. So you could do something like this:
+
+```javascript
+function myJsFunction() {
+  const transaction = Sentry.getCurrentHub().getScope().getTransaction();  
```

Comment:
Wouldn't this be a good case to use `withScope`?


```js
func f() {
  Sentry.withScope((scope) => {
    const transaction = scope.getTransaction();
    let span;
    if (transaction) {
      span = transaction.startChild({
        op: "encode",
        description: "parseAvatarImages"
      });
    } else {
      span = hub.startTransaction({
        name: "f",
        op: "encode",
        description: "parseAvatarImages"
      });
    }
    try {
      // Do work
    } finally {
      span.finish();
    }
  });
}

// did not test, may contain typos; typed in GitHub directly.
```

Reply:
`withScope` creates a new scope, but we could use `configureScope` here just fine imo.

```js
Sentry.configureScope(scope => {
  const transaction = scope.getTransaction();
  if (transaction) {
    let span = transaction.startChild({
      op: "encode",
      description: "parseAvatarImages"
    });
    // Do something
    span.finish();
  }
});